### ISSUE TITLE
Improved Clarity Contracts Coverage

### ISSUE THREAD

**Issue by jo-tm** [Trust-Machines/stacks-sbtc#525]

**Comment by jo-tm** [comment]
@MarvinJanssen do we need basic testing for peg-out-processor or is that on a branch? -> `sbtc-peg-out-            [0.0%]   0.0 %  0 / 30         0.0 %  0 / 5
  processor.clar`
@setzeus : do we need basic testing for peg-transfer or is that on a branch? -> `sbtc-peg-                [0.0%]   0.0 %  0 / 49         0.0 %  0 / 1
  transfer.clar`

**Comment by jo-tm** [comment]
@friedger : can write more tests for Stacking Pool https://github.com/Trust-Machines/stacks-sbtc/blob/main/sbtc-mini/tests/sbtc-stacking-pool_test.clar

**Comment by jo-tm** [comment]
@friedger : we need mocking things for Stacking Pools testing, now cannot be tested. #549 other alternatives is with standalone pox contract.

**Comment by jo-tm** [comment]
from @setzeus 💯

1. Getters/setters for pox-3
2. Deployment generation plans
3. Make it work on devnet
4. Generation script 2

**Comment by jo-tm** [comment]
WARNING: we have to ignore the `./tests/` folder in Coverage, now the numbers are lower due to this incorrect inclusion.

**Comment by MarvinJanssen** [comment]
Right, I tried to do that with codecov but they still included it. Probably an easy fix somewhere.

**Comment by AshtonStephens** [comment]
@jo-tm  is this work still getting done?

**Comment by jo-tm** [comment]
I think is not tracked here but tests are being added organically all the time.
